### PR TITLE
Generate code for Thrift Guantlet

### PR DESCRIPTION
**Pull request by abhinav**
This uses thriftrw-go to generate the code for the Thrift Gauntlet.

Implementation will come in a different PR.

### CODE REVIEW

**Comment by breerly** [comment]
This is exciting 💯

**Comment by abhinav** [comment]
This is ready for a look. @yarpc/golang

File: scripts/updateLicenses.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 set -e
 set -x
 
-python scripts/updateLicense.py $(go list -json $(glide nv) | jq -r '.Dir + "/" + (.GoFiles | .[])')
+python "$(dirname $0)"/updateLicense.py $(go list -json $(glide nv) | jq -r '.Dir + "/" + (.GoFiles | .[])')
```
Comment (breerly) [inline scripts/updateLicenses.sh]: does this change belong in this PR?

File: crossdock/thrift/gen.go [hunk 2]
```diff
@@ -21,3 +21,8 @@
 package echo_generator
 
 //go:generate thriftrw-go -yarpc echo.thrift
+//go:generate thriftrw-go -yarpc gauntlet.thrift
+
+// Run this last
+
+//go:generate ../../scripts/updateLicenses.sh
```
Comment (breerly) [inline crossdock/thrift/gen.go]: Did you write this file out by hand?

Comment (abhinav) [inline crossdock/thrift/gen.go]: Yes. This file is hand-written.

File: scripts/updateLicenses.sh (hunk 1 above)
Comment (abhinav) [inline scripts/updateLicenses.sh]: I mean, I could separate it but this is a trivial enough change that it doesn't really matter. The `updateLicenses` call in gen.go won't work without this change.

Comment (breerly) [inline scripts/updateLicenses.sh]: sure

**Comment by breerly** [comment]
The generated code looks pretty clean.

:shipit:

**Pull request merged**